lib/phlex/compiler/class_compiler.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Phlex::Compiler::ClassCompiler < Prism::Visitor
+	def initialize(compiler)
+		@compiler = compiler
+	end
+
+	def compile(node)
+		visit_all(node.child_nodes)
+	end
+
+	def visit_def_node(node)
+		Phlex::Compiler::MethodCompiler.new(@compiler).compile(node)
+	end
+
+	def visit_class_node(node)
+		nil
+	end
+
+	def visit_module_node(node)
+		nil
+	end
+
+	def visit_block_node(node)
+		nil
+	end
```

Comment:
We don’t want to visit other classes, modules or blocks since any method definitions here may be in the wrong scope.